superbench/benchmarks/micro_benchmarks/micro_base.py

```diff
@@ -21,8 +22,10 @@ def __init__(self, name, parameters=''):
         """
         super().__init__(name, parameters)
         self._benchmark_type = BenchmarkType.MICRO
+        # Set the binary directory.
+        self._binary_dir = os.getenv('SUPERBENCH_BINARY_DIR', '/usr/local/bin')
         # Command lines to launch the micro-benchmarks.
-        self.__commands = list()
+        self._commands = list()
```

Comment:
Why do we need to expose the commands?

Reply:
Here is not for exposing, just for inheriting. If we use self.__commands, the derived class can't access it.